.gitignore

```diff
@@ -57,3 +57,8 @@ cypress/screenshots
 
 # Redis related
 dump.rdb
+
+# Docusaurus build artifacts
+/docs/build
```

Comment:
Remove these entries. There is a gitignore file specific to docusaurus in the docs/ directory